src/parser/SGTypes.ts

```diff
@@ -161,6 +179,45 @@ export class SGScript extends SGTag {
             return super.transpileBody(state);
         }
     }
+
+    protected transpileAttributes(state: SGTranspileState): (string | SourceNode)[] {
```

Comment:
It might be better to have
```
    protected transpileAttributes(state: SGTranspileState, attributes: SGAttribute[]): (string | SourceNode)[] {
```
so the override could create a modified array in place and call the same super method.

Reply:
Good idea. I did that.